README.md

```diff
@@ -12,60 +12,58 @@ platform.
 
 # Supported Platforms #
 
-- [GCM](http://developer.android.com/guide/google/gcm/index.html) from google for android platform
-- [APNS](http://developer.apple.com/library/mac/#documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/ApplePushService/ApplePushService.html) from apple for iOS platform
-- [ADM](https://developer.amazon.com/sdk/adm.html) from amazon for Kindle tables
+- [GCM](http://developer.android.com/guide/google/gcm/index.html) from Google for the Android platform
+- [APNS](http://developer.apple.com/library/mac/#documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/ApplePushService/ApplePushService.html) from Apple for the iOS platform
+- [ADM](https://developer.amazon.com/sdk/adm.html) from Amazon for Kindle tablets
 
 # FAQ #
 
-- Q: A general push notification platform for all systems? How does this differ
-  from services like [urbanairship](http://urbanairship.com)?
-- A: [urbanairship](http://urbanairship.com) is a great service, and there're
-  other similar services available, like [openpush](http://openpush.im/),
-[notifica](https://notifica.re/), etc. All of them are wonderful services.
-However, [uniqush](http://uniqush.org) is different from them.
+- Q: Is this a general push notification platform for all systems? How does this differ
+  from services such as [Urban Airship](http://urbanairship.com)?
+- A: [Urban Airship](http://urbanairship.com) is a great service, and there are
```

Comment:
Could mention the following:

1. Some important features of uniqush are storing and automatically updating subscriptions, sending pushes, and maintaining persistent connections (if needed). (and providing a API that works for those platforms).
2. Other products have features uniqush doesn't have, such as scheduling(probably), analytics, client libraries for various mobile devices, a/b testing, etc.